packages/jobs/lib/crons/autoIdleDemo.ts

```diff
@@ -14,7 +14,7 @@ import {
     logger
 } from '@nangohq/shared';
 import { SpanTypes } from '@nangohq/shared';
-import tracer from '../tracer.js';
```

Comment:
I got the runner specific changes but why is this import change everywhere necessary?

Reply:
When we import tracer.js we init the tracer, because it's a singleton it's not necessary to import this tracer specifically. However because it's a singleton and we import files from others folder we had a double init before, to avoid this mistake again I think it's safer to not export the tracer.